modules/decision_reviews/lib/decision_reviews/notification_callbacks/va_notify_local_testing_readme.md

```diff
@@ -0,0 +1,310 @@
+## Testing DecisionReviews::FailureNotificationEmailJob + VA Notify Callbacks Locally
+
```

Comment:
I feel like I had to go through some headache in order to figure out how to test in staging last week for the release, and also locally to ensure the changes I made didn't break anything, so I compiled my notes into this guide for future reference.